src/poetry/console/application.py

```diff
@@ -367,6 +372,16 @@ def _default_definition(self) -> Definition:
             )
         )
 
+        definition.add_option(
+            Option(
+                "--project",
+                flag=False,
+                description=(
+                    "Specify path to Poetry project the command should work at."
```

Comment:
As above (and maybe we should consider a short flag like Git's `-C`?).

Reply:
I'm happy to add a short flag. But `-C` doesn't seem intuitive. Maybe `-p`? :thinking: